homework/english/Anastasia_Sheremet/lecture1/task1_grammar.md

```diff
@@ -0,0 +1,50 @@
+#Homework 1 Task 1 GRAMMAR
+
+Раскройте скобки, употребляя глаголы в Present Continuous или в Present Simple.
+
+    Is your mum working today?
+    Why this boys are talking now?
+    Where does he live?
+    Their mother cooks very well?
+    Lili is not rich, she does not drive a Mercedes.
+    Let´s go out. It is not raining now.
+    I have to go now. It is getting dark.
+    A young man usually stands at the window.
+    Ann usually makes doll dresses herself.
+    Is his wife having coffee at the moment?
+    Let’s go for a walk. It isn't raining.
+    Look! The boys are coming home.
+    Does Maila like Chinese food?
+    My grandmother does not work. She is on pension.
+    What is your brother watching now?
+    Where do you live?
+    Do You play the piano well?
+    Where is their brother swimming now?
+    What do they eat at breakfast?
+    Are they swimming at the moment?
+    Mike is wearing a tie today.
+    Maila does not watch TV every morning.
+    Listen! Blanca is singing in the bathroom.
+    Bea sometimes reads comics.
+    
+ MНайдите ошибки и исправьте их.
+
+    We not going to school today. => We are not going to school today.
```

Comment:
it's not process

Reply:
Please, pay attention to the marker 'today'. It's definitely correct.